website/src/components/table/table.tsx

```diff
@@ -14,6 +18,16 @@ const CSTable: React.FunctionComponent<{
   handleEditClick: (entryID: string) => void;
   handleDelete: (entryID: string) => void;
 }> = ({ data, handleEditClick, handleDelete }) => {
+  const signedInstate = useSelector(selectSignedInState);
+  const history = useHistory();
+
+  useEffect(() => {
+    // only redirect if not viewing another user
+    if (signedInstate.admin && isAdmin(signedInstate.userEmail)) {
+      history.replace("/admin");
+    }
+  });
+
```

Comment:
could you move this to the controller that wraps the table component? The ```table.tsx``` should be a purely UI component